src/c++/perf_analyzer/command_line_parser.cc

```diff
@@ -373,6 +374,13 @@ CLParser::Usage(const std::string& msg)
              "that the average latency is used to determine stability",
              18)
       << std::endl;
+  std::cerr << FormatMessage(
+                   " --serial-sequences: Enables serial sequence mode "
+                   "where a maximum of one request is outstanding at a time "
+                   "for any given sequence."
```

Comment:
I think this has no space after the period? Can you fix formatting?

Reply:
done